content/karpenter/index.md

```diff
@@ -253,14 +247,14 @@ The following best practices relate to deploying pods In a cluster using Karpent
 
 ### Follow EKS best practices for high availability
 
-If you need to run highly available applications, follow general EKS best practice [recommendations](https://aws.github.io/aws-eks-best-practices/reliability/docs/application/#recommendations). See [Topology Spread](https://karpenter.sh/preview/tasks/scheduling/#topology-spread) in Karpenter documentation for details on how to spread pods across nodes and zones. Use [Disruption Budgets](https://karpenter.sh/preview/tasks/deprovisioning/#disruption-budgets) to set the minimum available pods that need to be maintained, in case there are attempts to evict or delete pods.
+If you need to run highly available applications, follow general EKS best practice [recommendations](https://aws.github.io/aws-eks-best-practices/reliability/docs/application/#recommendations). See [Topology Spread](https://karpenter.sh/preview/tasks/scheduling/#topology-spread) in Karpenter documentation for details on how to spread pods across nodes and zones. Use [Disruption Budgets](https://karpenter.sh/docs/troubleshooting/#disruption-budgets) to set the minimum available pods that need to be maintained, in case there are attempts to evict or delete pods.
```

Comment:
```suggestion
If you need to run highly available applications, follow general EKS best practice [recommendations](https://aws.github.io/aws-eks-best-practices/reliability/docs/application/#recommendations). See [Topology Spread](https://karpenter.sh/docs/concepts/scheduling/#topology-spread) in Karpenter documentation for details on how to spread pods across nodes and zones. Use [Disruption Budgets](https://karpenter.sh/docs/troubleshooting/#disruption-budgets) to set the minimum available pods that need to be maintained, in case there are attempts to evict or delete pods.
```